rhoshift/utils/utils.py

```
@@ -0,0 +1,360 @@
+import logging
+import subprocess
```

Ruff (0.11.9)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)